hpOneView/resources/servers/migratable_vc_domains.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, connection):
 
     @staticmethod
     def make_migration_information(oaIpAddress, oaUsername, oaPassword, vcmUsername, vcmPassword,
-                                   iloLicenseType='OneView', enclosureGroupUri=None):
+									type = 'migratable-vc-domains', iloLicenseType='OneView', enclosureGroupUri=None):
```

Comment:
Adding type filed here is not a good idea.
Add DEFAULT_VALUES class variable to support type field for different versions.

Refer this file, https://github.com/HewlettPackard/python-hpOneView/blob/master/hpOneView/resources/networking/fc_networks.py